.github/actions/create-package-for-el-8/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+# Inputs
+
+PE_VERSION=${1}
+PLATFORM=${PLATFORM:-el-8}
+GITHUB_WORKSPACE=${GITHUB_WORKSPACE:-/workspace}
+SCRATCHDIR=${SCRATCHDIR:-/scratch}
+
+if [ -z "${PE_VERSION}" ]; then
+  echo 'ERROR: Did not recieve a valid PE version argument ($1)'
+  exit 1
+fi
+
+# Download PE and install required dependent packages
+
+mkdir -p "${SCRATCHDIR}/pe"
+curl -o "${SCRATCHDIR}/pe.tar.gz" "https://s3.amazonaws.com/pe-builds/released/${PE_VERSION}/puppet-enterprise-${PE_VERSION}-${PLATFORM}-x86_64.tar.gz"
+tar -C "${SCRATCHDIR}/pe" -xzf "${SCRATCHDIR}/pe.tar.gz" --strip-components 1
+
+cat > /etc/yum.repos.d/pe.repo <<EOF
+[pe]
+name=Puppet, Inc. PE Packages \$releasever - \$basearch
+baseurl=file://${SCRATCHDIR}/pe/packages/${PLATFORM}-x86_64
+enabled=True
+gpgcheck=1
+gpgkey=file://${SCRATCHDIR}/pe/packages/GPG-KEY-puppet
+        file://${SCRATCHDIR}/pe/packages/GPG-KEY-puppet-2025-04-06
+EOF
+
+yum install -y puppet-agent pe-puppet-enterprise-release pe-postgresql11-devel
+
+# Create the RPM
+
+# It is assumed/required that the puppet-data-service project is already
+# checked out in the GITHUB_WORKSPACE.
+mkdir -p "${GITHUB_WORKSPACE}"
+pushd "${GITHUB_WORKSPACE}"
+  make clean
+  make rpm
+  RPMFILE=$(ls -t *.rpm | head -1)
```

Comment:
Im a lot less fussed about this shellcheck recommendation but just because its suggested

In test.sh line 42:
RPMFILE=$(ls -t *.rpm | head -1)
^---------^ SC2012 (info): Use find instead of ls to better handle non-alphanumeric filenames.
^-- SC2035 (info): Use ./glob or -- glob so names with dashes won't become options.